### PR TITLE
docs: add SECURITY.md

### PR DESCRIPTION
**Pull request by Splaktar**
## PR Checklist
Please check that your PR fulfills the following requirements:
- [x] The commit message follows [our guidelines](https://github.com/angular/material/blob/master/.github/CONTRIBUTING.md#-commit-message-format)
- [x] Tests for the changes have been added or this is not a bug fix / enhancement
- [x] Docs have been added, updated, or were not required

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Enhancement
[x] Documentation content changes
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
There is no Security Policy defined in the repo.

Issue Number: 
N/A

## What is the new behavior?
There is a Security Policy defined in the repo.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
~support@angularjsmaterial.dev goes directly to me and has specific importance filters set up for it in Gmail.~

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+## Supported Versions
+
+| Version     | Supported          | Status                | Comments                             |
+| ----------- | ------------------ | --------------------- | ------------------------------------ |
+| 1.1.x       | :white_check_mark: | Active Development    |                                      |
+| <1.0.x      | :x:                |                       |                                      |
+
+## Reporting a Vulnerability
+
+Email us at [support@angularjsmaterial.dev](mailto:support@angularjsmaterial.dev) to report any potential security
```
Comment (jelbourn) [inline SECURITY.md:12]: Who owns this? AFAIK this should be security@angularjs.org for now.

Comment (Splaktar) [reply SECURITY.md:12]: Mentioned in the end of the description:
> support@angularjsmaterial.dev goes directly to me and has specific importance filters set up for it in Gmail.

I can do security@angularjs.org, I just don't have any access to it or any idea if the information would ever get to me.

Comment (jelbourn) [reply SECURITY.md:12]: Yeah, definitely security@angularjs.org ; we'll let you know if anything comes in (nothing has so far)

Comment (Splaktar) [reply SECURITY.md]: Updated.

**Pull request merged**